tests/sentry/mail/test_adapter.py

```diff
@@ -535,6 +598,8 @@ def test_notify_users_does_email(self, mock_logger, mock_func):
         self.assertEqual(notification.reference, group)
         assert notification.get_subject() == "BAR-1 - hello world"
 
+        assert notification.get_context()["snooze_alert"] is False
```

Comment:
Is this the only difference? If so, could you add a comment like you have in `src/sentry/notifications/notifications/rules.py` calling it out? Kinda hard to understand what this test is checking for otherwise